components/playlist/playlist_download_request_manager.cc

```diff
@@ -213,23 +230,45 @@ void PlaylistDownloadRequestManager::ProcessFoundMedia(
     auto* page_source = media.FindStringKey("pageSrc");
     auto* mime_type = media.FindStringKey("mimeType");
     auto* src = media.FindStringKey("src");
```

Comment:
for all the above, it says in chromium docs that `FindStringKey` is deprecated? https://source.chromium.org/chromium/chromium/src/+/main:base/values.h;l=751?q=findstringkey&ss=chromium

Reply:
You're right. I should have updated. Done, and thanks! :)